{"name": "test_mul[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 13", "trace": "self = <testing_calc1.test_calc_hw1.TestCalc object at 0x00000000037FE310>, begin_end = None, get_calc = <code_calc.calc.Calculator object at 0x0000000003841580>, get_mul_datas = [2, 5, 13]\n\n    @allure.story(\"测试乘法\")\n    @pytest.mark.run(order=3)\n    def test_mul(self, begin_end, get_calc, get_mul_datas):\n        # 调用 mul 方法\n        with allure.step(\"计算两个数相乘\"):\n            result = get_calc.mul(get_mul_datas[0], get_mul_datas[1])\n        # 处理浮点数的运算, 四舍五入\n        if isinstance(result, float):\n            result = round(result, 3)\n        # 断言\n>       assert result == get_mul_datas[2]\nE       assert 10 == 13\n\ntest_calc_hw1.py:111: AssertionError"}, "steps": [{"name": "计算两个数相乘", "status": "passed", "start": 1614442301690, "stop": 1614442301690}], "parameters": [{"name": "get_mul_datas", "value": "[2, 5, 13]"}], "start": 1614442301690, "stop": 1614442301690, "uuid": "b488eb37-e74f-43af-bbda-a685928bfc55", "historyId": "f1043c65a79b52a31a023540b1882b90", "testCaseId": "d8a4f27553c6baf2e2de8af026e44055", "fullName": "test_calc_hw1.TestCalc#test_mul", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "测试乘法"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "test_calc_hw1"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "CN-20190502UYEW"}, {"name": "thread", "value": "16908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_hw1"}]}